src/Spout/Reader/XLSX/Helper/CellValueFormatter.php

```diff
@@ -189,7 +189,17 @@ protected function formatExcelTimestampValue($nodeValue)
 
         // The value 1.0 represents 1900-01-01. Numbers below 1.0 are not valid Excel dates.
         if ($nodeValue < 1.0) {
-            return null;
+            // treat this value as time-only cell
```

Comment:
I'd create 2 functions to handle both cases (time and date). Also, can you move the check for < 1.0 (that can be replaced by < 1 I believe) before the leap year fix above? the leap year is only relevant in the case of dates.

So something like this:

```php
if ($nodeValue < 1) {
  return $this->formatExcelTimestampAsTimeValue($nodeValue);
} else {
  return $this->formatExcelTimestampAsDateValue($nodeValue);
}
```